docs/flashbots-auction/searchers/advanced/private-transaction.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: private transactions
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## How to send a single transaction to Flashbots
+
+If you want to send a single transaction to flashbots, without sending it as a bundle, you can use the `sendPrivateTransaction` method.
+
+This method has a few key advantages:
+
+1. This method attempts to send your transaction to miners on every block for a maximum of 25 blocks. No need to listen for the next block and re-send yourself. Just call `result.wait()` and check the result.
+2. Sending single transactions via this method is faster and more efficient than `sendBundle`. Transactions sent via this method are sent to our _bundler_, which creates bundles out of private transactions and sends them to miners. Ultimately, this means that your transaction will be mined sooner than if you were to send a single-transaction bundle with `sendBundle`.
```

Comment:
Not sure that (2) is true. There is a slight latency advantage in how quickly your tx would be sent as a bundle, but don't think it'll affect how quickly a tx will be mined in most cases.

Reply:
:heavy_check_mark: removed